config/manifests/patches/csvAnnotations.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: runtime-component.v0.0.0
   namespace: placeholder
   annotations:
-    containerImage: IMAGE
-    createdAt: CREATEDAT
\ No newline at end of file
+    containerImage: applicationstacks/operator:daily
```

Comment:
Please rebase and run the `make bundle` command. With the latest changes, this template file shouldn't be modified.